MetaMask/metamask-extension
# fix: UX: Remove legacy onboarding text

## **Description**

Removes the legacy onboarding text that was in place before the recent privacy notice update.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25404?quickstart=1)

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Go through onboarding
2. See new metametrics text
3. Look at code, don't see old onboarding text

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
